worklets/resources/import-tests.js

```diff
@@ -130,33 +130,33 @@ function runImportTests(worklet_type) {
     promise_test(t => {
         const kScriptURL = get_host_info().HTTPS_REMOTE_ORIGIN +
                            '/worklets/resources/empty-worklet-script.js';
-        return promise_rejects(t, new DOMException('', 'AbortError'),
-                               worklet.addModule(
-                                   '/common/redirect.py?location=' +
-                                   encodeURIComponent(kScriptURL)));
+        return promise_rejects_dom(t, 'AbortError',
+                                   worklet.addModule(
+                                     '/common/redirect.py?location=' +
+                                     encodeURIComponent(kScriptURL)));
```

Comment:
Nit-picking, but: this PR has reduced these lines from 4-space indent from the line above to 2-space.